types/transformer/transformer.go

```diff
@@ -46,6 +46,12 @@ type TransformerSpec struct {
 	OverrideSelector   labels.Selector                        `yaml:"-" json:"-"`
 	TemplatesDir       string                                 `yaml:"templates" json:"templates"` // Relative to yaml directory or working directory in image
 	Config             interface{}                            `yaml:"config" json:"config"`
+	InvokesByDefault   InvokesByDefault                       `yaml:"invokesByDefault" json:"invokesByDefault"`
+}
+
+// InvokesByDefault stores config to toggle default transformers
+type InvokesByDefault struct {
```

Comment:
And Keep this name consistent with the name we use in the plan.

Reply:
Shall we leave this the same? I mean we are using `invokesByDefault` in the plan file which goes well with `InvokedByDefaultTransformers` too

Reply:
changed it to `InvokedByDefault`